playbooks/configure-apt-sources.yml

```diff
@@ -151,16 +185,12 @@
         - apt_artifact_found | bool
         - apt_artifact_enabled | bool
 
-    - name: Remove extra sources
-      lineinfile:
-        dest: /etc/apt/sources.list
-        state: absent
-        regexp: "{{ item }}"
-      with_items:
-        - "^deb-src"
-        - "-backports"
-        - "-security"
-        - "-updates"
+    - name: Set the preferred repo to use to be the apt artifact repo
```

Comment:
This pin makes sense however I believe it should only be run when artifacts are enabled and  "apt_artifact_mode" is "loose". Otherwise we can assume the strict mode is in place and the apt sources have all been wiped out.

Reply:
The host OS may also have other repositories configured on them for other tooling. This ensures that our repository is still preferred above *any* other repository on the host.